### PR TITLE
Rename references to kubefed2 with kubefedctl

### PR DESCRIPTION
**Pull request by shashidharatd**
Fixes: https://github.com/kubernetes-sigs/federation-v2/issues/799

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
    kubefed2 disable now deletes the FederatedTypeConfig rather than set
```
Comment (font) [inline CHANGELOG.md]: Since this is unreleased it should probably be `kubefedctl`

Comment (marun) [reply CHANGELOG.md]: I wouldn't worry too much about that, it's necessary to edit the commit log anyway.

**Comment by irfanurrehman** [comment]
Some how `kubefedctl` still sounds like a mouthful....

**Comment by shashidharatd** [comment]
> Some how kubefedctl still sounds like a mouthful....

This is the next best thing after `kfctl`. And seems to have consensus. Do we need some more time to deliberate?

**Comment by marun** [comment]
@shashidharatd It seems like `kfedctl` is still in the running.  Let's discuss at today's meeting.

**Comment by shashidharatd** [comment]
so `kubefedctl` it is. have rebased and resolved conflict. @marun, @font, please take a cursory look once again. Thanks !

**Comment by font** [comment]
Thanks @shashidharatd !

**Pull request merged**